### PR TITLE
Add ability to stop non-queue experiments from the table's context menu

### PR DESCRIPTION
**Pull request by mattseddon**
This PR adds the ability to stop experiments which are running in the workspace from the table's context menu.

### Demo

https://user-images.githubusercontent.com/37993418/217762449-f51c83c7-2499-4f22-82a2-f49e96f8927d.mov

### CODE REVIEW

**Review by sroy3 — APPROVED** [review]

File: extension/src/experiments/processExecution/index.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  const pid = await getPidFromFile(join(dvcRoot, EXP_RWLOCK_FILE))
+  if (!pid) {
+    return
+  }
+  if (!(await processExists(pid))) {
+    return
+  }
+
+  void Toast.showOutput(
+    stopProcesses([pid]).then(stopped =>
```
Comment (julieg18) [inline extension/src/experiments/processExecution/index.ts:17]: Spotted something weird while testing though it's probably just a local issue (wouldn't be the first time 😅). I was on the `main` branch but when I stopped a running experiment and went back to my terminal, my terminal was checked out to a different position. 

https://user-images.githubusercontent.com/43496356/217856044-4aa3ec67-7d8c-4a54-a96b-e546ff032a22.mov

**Review by julieg18 — APPROVED** [review]
Great work!

File: extension/src/experiments/processExecution/index.ts (hunk 1 above)
Comment (mattseddon) [reply extension/src/experiments/processExecution/index.ts:17]: This is what happens when you stop a checkpoint experiment. HEAD is being moved during the experiment.

File: webview/src/experiments/components/table/RowContextMenu.tsx [hunk 2]
```diff
@@ -219,10 +222,10 @@ const getSingleSelectMenuOptions = (
       !hasRunningExperiment
     ),
     experimentMenuOption(
-      [id],
+      [{ executor, id }],
```
[the remark this answers is not in the document]
Comment (mattseddon) [reply webview/src/experiments/components/table/RowContextMenu.tsx:225]: I'll get rid of the separator.

**Pull request merged**